pipeline/metadata/quincy.yaml

```diff
@@ -0,0 +1,61 @@
+#=====================================================================================
+# Metadata file for quincy upstream testingn.
+# Single file to specify test suites to be executed for all regression (sanity) and schedule tests
+# as per defined in each stages.
+# Each stage will execute in sequential pattern.
+# Contains all Default parameter used for Execution when no override is specified.
+# parameter required to create a similar custom file is suite name, suite yaml file, global configuration file,
+# platform, rhbuild, inventory and metadata information like frequency of execution, tier, cloud type, functional group and stage.
+#=====================================================================================
+
+- name: "Beginning Red Hat Cephadm testing with upstream build"
```

Comment:
name: "Beginning Red Hat Cephadm testing with upstream build" 

please reword to generic name.

Reply:
Made changes as per comments